cert-manager/overlays/moc/infra/openshift-ingress-issuer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: cert-manager.io/v1
```

Comment:
Why not just create a `ClusterIssuer` rather than two identically configured namespaced-issuers?

Reply:
Yeah, I suppose I am trying to be consistent with the way we deployed it on smaug, if I change it to `clusterissuer` then we''d have ClusterIssuer's in `cluster-scope` and then `issuers` in `cert-manager` app,

Reply:
if you set up a `ClusterIssuer` in a cluster, is it available to be used in any namespace?
If yes, I think we should stick to using single issuers to prevent other users from being able to use this letsencrypt account

Reply:
How about in order to maintain consistency we go with `issuer` for now, and split off a separate discussion in a different issue, we'll have to reorganize the cert-manager directory at some point anyway as it's a mess right now.